<launch>
    <arg name="ns" default="/" />
    <group ns="$(arg ns)">
        <rosparam>
            pose_topic: "robot_poses"
                        <!-- pose_topic: "filtered_poses" -->
            wheel_radius: 0.01
            wheel_distance: 0.0451
            naive_burst_and_coast:
                rate: 30
        </rosparam>

        <!-- Thymio values -->
        <!-- <rosparam>
            pose_topic: "robot_poses"
            wheel_radius: 0.022
            wheel_distance: 0.0475      
        </rosparam> -->

        <node name="naive_burst_and_coast_node" pkg="bobi_control" type="naive_burst_and_coast_node" output="screen"/>
    </group>
</launch>